src/rpc/mining.cpp

```diff
@@ -210,7 +210,10 @@ static RPCHelpMan generatetodescriptor()
 {
     return RPCHelpMan{
         "generatetodescriptor",
-        "\nMine blocks immediately to a specified descriptor (before the RPC call returns)\n",
+        "\nMine blocks immediately to a specified descriptor (before the RPC call returns)\n"
```

Comment:
```suggestion
        "Mine blocks immediately to a specified descriptor (before the RPC call returns)\n"
```

While touching this, there is no need to keep the `\n`s in the beginning and end.

Reply:
suggestions for each of these helps
- period at end of sentences
- shorten the longer line
- maybe s/maxtries/`maxtries`/
- omit newlines at start and end
 
```diff
-        "\nMine blocks immediately to a specified descriptor (before the RPC call returns)\n"
-        "Only maxtries hash attempts will be performed. If no valid block can be found among those,\n"
-        "an error will be returned. This RPC is only intended for testing purposes, as it is highly inefficient and extremely\n"
-        "unlikely to succeed on mainnet\n",
+        "Mine blocks immediately to a specified descriptor (before the RPC call returns).\n"
+        "Only `maxtries` hash attempts will be performed. If no valid block can be found among those,\n"
+        "an error will be returned. This RPC is only intended for testing purposes, as it is highly\n"
+        "inefficient and extremely unlikely to succeed on mainnet.",
```
output
```
Mine blocks immediately to a specified descriptor (before the RPC call returns).
Only `maxtries` hash attempts will be performed. If no valid block can be found among those,
an error will be returned. This RPC is only intended for testing purposes, as it is highly
inefficient and extremely unlikely to succeed on mainnet.
```

Reply:
If this turns into a pull to rework the docs, the "immediately" and "(before the RPC call returns)" can be removed. It should be clear from the return value (the block hash) that the block was generated before the has was returned.